### ISSUE TITLE
Problems with network connections after Home Assistant Blue has been updated to OS 10.0

### ISSUE THREAD

**Issue by Podler** [home-assistant/operating-system#2511]
### Describe the issue you are experiencing

Since updating to OS 10.0 on Home Assistant Blue Hardware the system experiences several issues with network connections. 

For example KNX-Ultimate for Node Red (https://flows.nodered.org/node/node-red-contrib-knx-ultimate) looses networking connection to KNX IP Routers after several minutes. The Synology integration experiences problems to display camera live views and so on.

### What operating system image do you use?

~~generic-x86-64 (Generic UEFI capable x86-64 systems)~~ see text above

### What version of Home Assistant Operating System is installed?

10.0

### Did you upgrade the Operating System.

Yes

### Steps to reproduce the issue

Occurs again shortly after reboot.

### Anything in the Supervisor logs that might be useful for us?

```txt
23-04-25 10:11:07 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:07 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:07 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:07 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:11:09 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:09 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:09 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:11:13 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:14 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:11:34 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:11:34 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:14:30 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:14:30 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:14:39 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:14:39 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:14:44 INFO (MainThread) [supervisor.auth] Auth request from 'a0d7b954_nodered' for 'Thomas'
23-04-25 10:14:44 INFO (MainThread) [supervisor.auth] Successful login for 'Thomas'
23-04-25 10:15:50 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.NO_CURRENT_BACKUP/ContextType.SYSTEM
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.IPV4_CONNECTION_PROBLEM/ContextType.SYSTEM
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.MULTIPLE_DATA_DISKS/ContextType.SYSTEM
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-04-25 10:19:47 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
23-04-25 10:19:48 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-04-25 10:19:48 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
23-04-25 10:19:48 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
23-04-25 10:19:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-04-25 10:20:54 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:25:58 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:31:02 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:36:06 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:41:10 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:46:14 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:49:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-04-25 10:51:18 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 10:56:22 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:01:26 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:06:30 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:11:34 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:16:38 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.NO_CURRENT_BACKUP/ContextType.SYSTEM
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.IPV4_CONNECTION_PROBLEM/ContextType.SYSTEM
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.MULTIPLE_DATA_DISKS/ContextType.SYSTEM
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
23-04-25 11:19:48 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
23-04-25 11:19:48 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
23-04-25 11:19:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-04-25 11:21:42 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:26:46 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:31:50 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:36:54 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:41:58 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:46:28 INFO (MainThread) [supervisor.host.info] Updating local host information
23-04-25 11:46:29 INFO (MainThread) [supervisor.host.services] Updating service information
23-04-25 11:46:29 INFO (MainThread) [supervisor.host.network] Updating local network information
23-04-25 11:46:29 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
23-04-25 11:46:29 INFO (MainThread) [supervisor.host.manager] Host information reload completed
23-04-25 11:47:02 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:49:49 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-04-25 11:52:06 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
23-04-25 11:57:10 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)
```

### Anything in the Host logs that might be useful for us?

```txt
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5170] device (eth0): state change: config -> ip-config (reason 'none', sys-iface-state: 'managed')
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5180] policy: set 'Supervisor eth0' (eth0) as default for IPv4 routing and DNS
Apr 25 01:19:44 homeassistant systemd-resolved[425]: eth0: Bus client set default route setting: yes
Apr 25 01:19:44 homeassistant systemd-resolved[425]: eth0: Bus client set MulticastDNS setting: yes
Apr 25 01:19:44 homeassistant systemd-resolved[425]: eth0: Bus client set DNS server list to: 192.168.10.10
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5266] device (eth0): state change: ip-config -> ip-check (reason 'none', sys-iface-state: 'managed')
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5308] device (eth0): state change: ip-check -> secondaries (reason 'none', sys-iface-state: 'managed')
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5313] device (eth0): state change: secondaries -> activated (reason 'none', sys-iface-state: 'managed')
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5323] manager: NetworkManager state is now CONNECTED_SITE
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.5328] device (eth0): Activation: successful, device activated.
Apr 25 01:19:44 homeassistant NetworkManager[427]: <info>  [1682385584.6020] manager: NetworkManager state is now CONNECTED_GLOBAL
Apr 25 01:19:44 homeassistant os-agent[169]: INFO: 2023/04/25 01:19:44 apparmor.go:45: Load AppArmor profile '/mnt/data/supervisor/apparmor/hassio-supervisor'.
Apr 25 01:19:44 homeassistant os-agent[169]: INFO: 2023/04/25 01:19:44 apparmor.go:53: Load profile '/mnt/data/supervisor/apparmor/hassio-supervisor':
Apr 25 01:19:54 homeassistant systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Apr 25 01:20:16 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Apr 25 01:20:16 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Apr 25 01:20:16 homeassistant kernel: kauditd_printk_skb: 42 callbacks suppressed
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.219:353): prog-id=111 op=UNLOAD
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.219:354): prog-id=110 op=UNLOAD
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.219:355): prog-id=109 op=UNLOAD
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.267:356): prog-id=108 op=UNLOAD
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.267:357): prog-id=107 op=UNLOAD
Apr 25 01:20:16 homeassistant kernel: audit: type=1334 audit(1682385616.267:358): prog-id=106 op=UNLOAD
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.147:359): prog-id=112 op=LOAD
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.147:360): prog-id=113 op=LOAD
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.147:361): prog-id=114 op=LOAD
Apr 25 03:26:26 homeassistant systemd[1]: Starting Hostname Service...
Apr 25 03:26:26 homeassistant systemd[1]: Started Hostname Service.
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.439:362): prog-id=115 op=LOAD
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.439:363): prog-id=116 op=LOAD
Apr 25 03:26:26 homeassistant kernel: audit: type=1334 audit(1682393186.439:364): prog-id=117 op=LOAD
Apr 25 03:26:26 homeassistant systemd[1]: Starting Time & Date Service...
Apr 25 03:26:26 homeassistant systemd[1]: Started Time & Date Service.
Apr 25 03:26:56 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.501:365): prog-id=114 op=UNLOAD
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.501:366): prog-id=113 op=UNLOAD
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.501:367): prog-id=112 op=UNLOAD
Apr 25 03:26:56 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.773:368): prog-id=117 op=UNLOAD
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.773:369): prog-id=116 op=UNLOAD
Apr 25 03:26:56 homeassistant kernel: audit: type=1334 audit(1682393216.773:370): prog-id=115 op=UNLOAD
Apr 25 05:33:06 homeassistant kernel: audit: type=1334 audit(1682400786.870:371): prog-id=118 op=LOAD
Apr 25 05:33:06 homeassistant kernel: audit: type=1334 audit(1682400786.870:372): prog-id=119 op=LOAD
Apr 25 05:33:06 homeassistant kernel: audit: type=1334 audit(1682400786.870:373): prog-id=120 op=LOAD
Apr 25 05:33:06 homeassistant systemd[1]: Starting Hostname Service...
Apr 25 05:33:07 homeassistant systemd[1]: Started Hostname Service.
Apr 25 05:33:07 homeassistant kernel: audit: type=1334 audit(1682400787.134:374): prog-id=121 op=LOAD
Apr 25 05:33:07 homeassistant kernel: audit: type=1334 audit(1682400787.134:375): prog-id=122 op=LOAD
Apr 25 05:33:07 homeassistant kernel: audit: type=1334 audit(1682400787.134:376): prog-id=123 op=LOAD
Apr 25 05:33:07 homeassistant systemd[1]: Starting Time & Date Service...
Apr 25 05:33:07 homeassistant systemd[1]: Started Time & Date Service.
Apr 25 05:33:37 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.220:377): prog-id=120 op=UNLOAD
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.220:378): prog-id=119 op=UNLOAD
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.220:379): prog-id=118 op=UNLOAD
Apr 25 05:33:37 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.440:380): prog-id=123 op=UNLOAD
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.440:381): prog-id=122 op=UNLOAD
Apr 25 05:33:37 homeassistant kernel: audit: type=1334 audit(1682400817.440:382): prog-id=121 op=UNLOAD
Apr 25 06:19:51 homeassistant dockerd[500]: time="2023-04-25T06:19:51.117568658Z" level=warning msg="[resolver] failed to read from DNS server: 192.168.10.10:53, query: ;checkonline.home-assistant.io.\tIN\t A" error="read udp 172.30.232.2:42893->192.168.10.10:53: i/o timeout"
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.636:383): prog-id=124 op=LOAD
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.636:384): prog-id=125 op=LOAD
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.636:385): prog-id=126 op=LOAD
Apr 25 07:39:47 homeassistant systemd[1]: Starting Hostname Service...
Apr 25 07:39:47 homeassistant systemd[1]: Started Hostname Service.
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.936:386): prog-id=127 op=LOAD
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.936:387): prog-id=128 op=LOAD
Apr 25 07:39:47 homeassistant kernel: audit: type=1334 audit(1682408387.936:388): prog-id=129 op=LOAD
Apr 25 07:39:47 homeassistant systemd[1]: Starting Time & Date Service...
Apr 25 07:39:48 homeassistant systemd[1]: Started Time & Date Service.
Apr 25 07:40:17 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.046:389): prog-id=126 op=UNLOAD
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.046:390): prog-id=125 op=UNLOAD
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.046:391): prog-id=124 op=UNLOAD
Apr 25 07:40:18 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.262:392): prog-id=129 op=UNLOAD
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.262:393): prog-id=128 op=UNLOAD
Apr 25 07:40:18 homeassistant kernel: audit: type=1334 audit(1682408418.262:394): prog-id=127 op=UNLOAD
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.543:395): prog-id=130 op=LOAD
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.543:396): prog-id=131 op=LOAD
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.543:397): prog-id=132 op=LOAD
Apr 25 09:46:28 homeassistant systemd[1]: Starting Hostname Service...
Apr 25 09:46:28 homeassistant systemd[1]: Started Hostname Service.
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.827:398): prog-id=133 op=LOAD
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.827:399): prog-id=134 op=LOAD
Apr 25 09:46:28 homeassistant kernel: audit: type=1334 audit(1682415988.827:400): prog-id=135 op=LOAD
Apr 25 09:46:28 homeassistant systemd[1]: Starting Time & Date Service...
Apr 25 09:46:29 homeassistant systemd[1]: Started Time & Date Service.
Apr 25 09:46:58 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Apr 25 09:46:58 homeassistant kernel: audit: type=1334 audit(1682416018.936:401): prog-id=132 op=UNLOAD
Apr 25 09:46:58 homeassistant kernel: audit: type=1334 audit(1682416018.936:402): prog-id=131 op=UNLOAD
Apr 25 09:46:58 homeassistant kernel: audit: type=1334 audit(1682416018.936:403): prog-id=130 op=UNLOAD
Apr 25 09:46:59 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Apr 25 09:46:59 homeassistant kernel: audit: type=1334 audit(1682416019.120:404): prog-id=135 op=UNLOAD
Apr 25 09:46:59 homeassistant kernel: audit: type=1334 audit(1682416019.120:405): prog-id=134 op=UNLOAD
Apr 25 09:46:59 homeassistant kernel: audit: type=1334 audit(1682416019.120:406): prog-id=133 op=UNLOAD
Apr 25 09:59:32 homeassistant kernel: audit: type=1334 audit(1682416772.877:407): prog-id=136 op=LOAD
Apr 25 09:59:32 homeassistant systemd-timesyncd[485]: Network configuration changed, trying to establish connection.
Apr 25 09:59:32 homeassistant systemd-timesyncd[485]: Contacted time server 162.159.200.1:123 (time.cloudflare.com).
Apr 25 09:59:32 homeassistant systemd[1]: Started Journal Gateway Service.
Apr 25 09:59:33 homeassistant systemd-journal-gatewayd[90543]: microhttpd: MHD_OPTION_EXTERNAL_LOGGER is not the first option specified for the daemon. Some messages may be printed by the standard MHD logger.
```

## System Information

version | core-2023.4.6
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.10
os_name | Linux
os_version | 6.1.24
arch | aarch64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 18. Mai 2023 um 02:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | false
remote_server | eu-central-1-13.ui.nabu.casa
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 10.0
-- | --
update_channel | stable
supervisor_version | supervisor-2023.04.1
agent_version | 1.5.1
docker_version | 23.0.3
disk_total | 113.9 GB
disk_used | 9.1 GB
healthy | true
supported | true
board | odroid-n2
supervisor_api | ok
version_api | ok
installed_addons | Node-RED (14.1.5), File editor (5.6.0), Terminal & SSH (9.7.0), Samba share (10.0.1), Mosquitto broker (6.2.0)

</details>

<details><summary>Husqvarna Automower</summary>

can_reach_server | ok
-- | --

</details>

<details><summary>Dashboards</summary>

dashboards | 11
-- | --
resources | 0
views | 11
mode | storage

</details>

<details><summary>Miele</summary>

component_version | 0.1.12
-- | --
reach_miele_cloud | ok

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 15. April 2023 um 12:31
-- | --
current_recorder_run | 24. April 2023 um 18:38
estimated_db_size | 643.39 MiB
database_engine | sqlite
database_version | 3.38.5

</details>

<details><summary>Spotify</summary>

api_endpoint_reachable | ok
-- | --

</details>

**Comment by agners** [comment]
Can you check and share the logs of the Node RED add-on?

To rule out any other issue, can you try to downgrade again and see if that indeed fixes the problem:

```
ha os update --version=9.5
```

**Comment by Podler** [comment]
I updated the system to HA OS 10.1 in the meantime. Issue is still there and several problems with network connectivity persist. Trying to downgrade to 9.5 now, test the system and than try the update to 10.1 again. Will keep this tread updated after that.

**Comment by Podler** [comment]
Downgrade to HA OS 9.5 solves the problem. Network connectivity stable again. I am going to monitor the system for some hours and than try the 10.1 Update again.

**Comment by Podler** [comment]
The direct update from Home Assistant OS 9.5 to 10.1 did not recreate the networking issue. Maybe something went wrong with the first upgrade to 10.0 or a migration bug was solved in 10.1. Thank you very much for your support and your indispensable knx ultimate node red integration! Regards, Thomas.
